client/components/mma/cancel/supporterplus/SupporterplusCancellationReasons.tsx

```diff
@@ -84,9 +84,21 @@ export const supporterplusCancellationReasons: CancellationReason[] = [
 		linkLabel: 'I no longer read the Guardian enough to justify my support',
 		alternateFeedbackIntro: standardAlternateFeedbackIntro,
 	},
+];
+
+export const otherCancellationReason: CancellationReason[] = [
 	{
 		reasonId: 'mma_other',
 		linkLabel: 'Another reason (please specify)',
 		alternateFeedbackIntro: standardAlternateFeedbackIntro,
 	},
 ];
+
+const shuffledArray = [...supporterplusCancellationReasons].sort(
+	() => 0.5 - Math.random(),
+);
```

Comment:
as above this produces quite a biased distribution of ordering, e.g. the first element in the initial list is more than twice as likely to remain first than the second element
See discussion here:
https://github.com/guardian/manage-frontend/pull/1415/files#r1846461396